README.md

```diff
@@ -1,2 +1,35 @@
 # github-actions
 A collection of reusable and composite Github Actions
+
+## maven/build-scan-save
+A Composite action to save an unpublished Maven Build Scan®.
+
+The action uploads the Build Scan® (if present) as a workflow artifact with name `maven-build-scan-data`.
```

Comment:
Suggested content:

```
The action saves unpublished build scan data as a workflow artifact with name `maven-build-scan-data`, which can then be published in a dependent workflow.

Use this action in your existing pull-request workflows to allow build scans to be published. Since these workflows are running in an untrusted context, they do not have access to the required secrets to publish the build scans directly.

Since the Gradle Enterprise Maven Extension only saves the build scan data for the most recent Maven execution, a step using this action must be inserted after each Maven execution step in the workflow.
```

Reply:
We'll also need a high-level overview describing the way the 3 actions work together, but we can add this at the end.